src/applications/check-in/utils/pilotFeatures/index.js

```diff
@@ -0,0 +1,82 @@
+import wilkesBarre from './travelClaim/wilkesBarre.json';
+// import salem from './travelClaim/salem.json';
+
+const devStations = (withDev = true, withStage = true, withLocal = true) => {
+  const dev = {
+    '500': {},
+  };
+  const stage = {
+    '530': {},
+  };
+  const local = {
+    '0001': {
+      clinics: ['0001'],
+    },
+  };
+
+  return {
+    ...(withDev ? dev : {}),
+    ...(withStage ? stage : {}),
+    ...(withLocal ? local : {}),
+  };
+};
+
+const pilotFeatures = {
+  fileTravelClaim: {
+    pilotStations: {
+      ...devStations(),
+      ...wilkesBarre,
+      // Week 1 658
+      // ...salem,
+      // Week 2 565
+      // ...fayetteville,
+      // Week 3 558
+      // ...durham,
+      // Week 4 652
+      // ...hunterHolmes,
+      // Week 5 590
+      // ...hampton,
+      // Week 6 659
+      // ...salisbury,
+      // Week 7 637
+      // ...asheville,
+      // Week 8
+      // VISN 15
+    },
+  },
+  fortyFiveMinuteText: {
+    pilotStations: {
+      ...devStations(),
+    },
+  },
+};
+
+/**
+ * @param {Object} appointment
+ * @param {string} pilotFeature
+ * @returns {bool}
+ * */
+
+const isInPilot = ({ appointment, pilotFeature }) => {
```

Comment:
This is by design. It only evaluates very specifically to an appointment. That way you can decide in the implementation if you want one appointment to rule them all or if you want to evaluate individually. Another thing is that this could be used in the single appointment context, like a detail page where we may be piloting a feature. This is the reason this isn't a global written in redux.

Reply:
Yeah that makes sense I was thinking about it all wrong.